pom.xml

```diff
@@ -425,10 +428,15 @@
         <version>1.12.0</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.github.olivergondza</groupId>
+        <artifactId>maven-jdk-tools-wrapper</artifactId>
+        <version>0.1</version>
```

Comment:
I'm confused as to why certain versions are properties and others are not.

Reply:
That particular section is in dependencyManagement?

Reply:
I am just curious why certain versions in `dependencyManagement` get a property defined (e.g., `slf4jVersion`), but most dependencies don't have one.

Reply:
If the version needs to be reused in multiple places - if you look at slf4jVersion its also used in the javadoc configuration